addons/stock/models/stock_move.py

```diff
@@ -289,7 +290,7 @@ def _compute_reserved_availability(self):
         and is represented by the aggregated `product_qty` on the linked move lines. If the move
         is force assigned, the value will be 0.
         """
-        result = {data['move_id'][0]: data['product_qty'] for data in 
```

Comment:
Please configure your editor to avoid trimming whitespaces.

Reply:
It does not do that automatically but I triggered the formatting, an old habbit ... sorry I'll pay more attention in the future.